pkg/netconf/netconf_linux.go

```diff
@@ -157,7 +157,7 @@ func populateDefault(netCfg *NetworkConfig) {
 	}
 }
 
-func ApplyNetworkConfigs(netCfg *NetworkConfig, userSetHostname, userSetDNS bool) (bool, error) {
+func ApplyNetworkConfigs(netCfg *NetworkConfig, userSetHostname, userSetDNS bool, dhcpTimeout string) (bool, error) {
```

Comment:
`DHCPTimeout` is a field of `NetworkConfig`, do we still need to pass `dhcpTimeout` here?